lib/ansible/modules/network/junos/junos_user.py

```diff
@@ -250,7 +250,7 @@ def main():
     """ main entry point for module execution
     """
     argument_spec = dict(
-        users=dict(type='list', aliases=['collection']),
+        users=dict(type='list', aliases=['collection', 'aggregate']),
```

Comment:
swap `aggregate` and `users` so the argument is `aggregate` and the alias is `users`